bias: [-2.795981410299707, 5.56537795607384, -9.26261480175765, 2.857270304121247,
  5.101891725502269, 0.47273286554110444]
gripper_com_frame_id: left_measure
gripper_com_pose: [0.4050367366091169, 0.054558844182947534, 0.35794147457097386,
  0.0, 0.0, 0.0]
gripper_mass: 0.31797013029216364
